test/model_kit_test.jl

```diff
@@ -165,16 +165,16 @@
         @test evaluate_and_jacobian!(u, U, H, v, s) === nothing
         @test u ≈ û
         @test U ≈ Û
-
-        û = FD.grad(fdm, s -> H_test(v, s), s)[1]
-        @test dt(H, v, s) ≈ û
-        @test dt(ModelKit.interpreted(H), v, s) ≈ û
-        @test dt!(u, H, v, s) ≈ û
-        u .= 0
-        U .= 0
-        @test jacobian_and_dt!(U, u, H, v, s) === nothing
-        @test u ≈ û
-        @test U ≈ Û
+        #
+        # # û = FD.grad(fdm, s -> H_test(v, s), s)[1]
+        # @test dt(H, v, s) ≈ û
+        # @test dt(ModelKit.interpreted(H), v, s) ≈ û
+        # @test dt!(u, H, v, s) ≈ û
+        # u .= 0
+        # U .= 0
+        # @test jacobian_and_dt!(U, u, H, v, s) === nothing
+        # @test u ≈ û
+        # @test U ≈ Û
```

Comment:
You could use `@test_broken` or `@test_skip` instead ([docs](https://docs.julialang.org/en/v1/stdlib/Test/#Broken-Tests-1)), so you keep being reminded of them.